src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -38,20 +39,30 @@ export class DownloadCLIClientComponent implements OnInit {
       (resultFromApi: Metadata) => {
         apiVersion = resultFromApi.version;
         this.dockstoreVersion = `${apiVersion}`;
-        this.downloadCli = `https://github.com/dockstore/dockstore/releases/download/${apiVersion}/dockstore`;
-        this.metadataService
-          .getRunnerDependencies(apiVersion, '3', 'cwltool', 'json')
-          .pipe(finalize(() => this.generateMarkdown()))
-          .subscribe(
-            (json: any) => {
-              if (json) {
-                this.cwltoolVersion = json.cwltool;
-              }
-            },
-            (err) => {
-              console.log('Unable to retrieve requirements.txt file.');
+        this.metadataService.getCliVersion().subscribe(
+          (json: any) => {
+            if (json) {
+              this.dockstoreCliVersion = json;
             }
-          );
+            this.downloadCli = `https://github.com/dockstore/dockstore-cli/releases/download/${this.dockstoreCliVersion}/dockstore`;
```

Comment:
Looking at how you're constructing this, why not instead have the web service endpoint return the complete url (or urls if you also return the unstable)?

Reply:
now returns complete URL